### ISSUE TITLE
FR: parent selection dropdown

### ISSUE THREAD

**Issue by EatThePath** [Baezon/pof-tools#21]
On paths there's a parent field. It'd be handy to have a list of valid parents to pick from.

**Comment by Baezon** [comment]
Good idea, will need some investigation to ensure that that is necessary for them to function. My fear is that no parent is a valid and usable option, although I suppose that could also be in the dropdown as a special choice...
